requirements.txt

```diff
@@ -1,3 +1,7 @@
 requests
-google_trans_new
-RAKEkeywords
\ No newline at end of file
+quebra_frases
+RAKEkeywords~=0.2.0
+neon-lang-plugin-libretranslate
```

Comment:
it does, this is the default plugin, its actually an upstream dependency

Reply:
Should this dependency be included in opm then instead of here?

Reply:
once the solvers are package this will be updated, this should be set where translation is used (neon_solvers main package)

this is just a workaround so PR can be tested with git installs of the solvers